internal/attestation/measurements/measurements.go

```diff
@@ -12,33 +12,53 @@ import (
 	"crypto/sha256"
 	"encoding/base64"
 	"encoding/hex"
+	"errors"
 	"fmt"
 	"io"
 	"net/http"
 	"net/url"
 
-	"github.com/edgelesssys/constellation/v2/internal/attestation/vtpm"
 	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
 	"github.com/edgelesssys/constellation/v2/internal/sigstore"
+	"github.com/google/go-tpm/tpmutil"
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	// PCRIndexClusterID is a PCR we extend to mark the node as initialized.
+	// The value used to extend is a random generated 32 Byte value.
+	PCRIndexClusterID = tpmutil.Handle(15)
+	// PCRIndexOwnerID is a PCR we extend to mark the node as initialized.
+	// The value used to extend is derived from Constellation's master key.
+	// TODO: move to stable, non-debug PCR before use.
+	PCRIndexOwnerID = tpmutil.Handle(16)
+)
+
 // Measurements are Platform Configuration Register (PCR) values.
-type Measurements map[uint32][]byte
+type Measurements map[uint32]Measurement
+
+// Measurement wraps expected PCR value and whether it is enforced.
+type Measurement struct {
+	Expected [32]byte `yaml:"expected"`
+	Enforced bool     `yaml:"enforced"`
```

Comment:
We would need to ensure that this is a required field. Alternatively, we could "invert" the field to something like `WarnOnly` so the default `false` is secure.